pkg/jx/cmd/step_create_task.go

```diff
@@ -316,6 +316,11 @@ func (o *StepCreateTaskOptions) Run() error {
 
 	if o.NoApply {
 		o.BuildNumber = "1"
+		if o.OutDir == "" {
+			log.Infof("No output directory provided, using %s\n", util.ColorInfo("out"))
```

Comment:
OOI, why not make `out` the default where the options are defined - https://github.com/jenkins-x/jx/blob/98a094d98c282fe4d42e533b0abda3b078538632/pkg/jx/cmd/step_create_task.go#L148? 

Or maybe even require it to be set and abort if not. 

Just trying to understand the reasoning for this option. ATM, I would think the first option would be the most consistent one.